Index: app/src/main/java/com/example/tourguideegyptian/MostVisitedPlacesActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/tourguideegyptian/MostVisitedPlacesActivity.java	(date 1568420617266)
+++ app/src/main/java/com/example/tourguideegyptian/MostVisitedPlacesActivity.java	(date 1568420617266)
@@ -0,0 +1,14 @@
+package com.example.tourguideegyptian;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class MostVisitedPlacesActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_most_visited_places);
+    }
+}
